components/playground/instance/tiflash.go

```diff
@@ -96,7 +104,28 @@ func (inst *TiFlashInstance) Start(ctx context.Context, version v0manifest.Versi
 
 	// Wait for PD
 	pdClient := api.NewPDClient(endpoints, 10*time.Second, nil)
-	enablePlacementRules, err := json.Marshal(replicateConfig{
+	// set low-space-ratio to 1 to avoid low disk space
+	lowSpaceRatio, err := json.Marshal(scheduleConfig{
+		LowSpaceRatio: 1,
```

Comment:
In PD's [document](Maximum value: less than 1), it is "less than 1", but not "less than or equal 1".
Maybe set it to 0.99?

Reply:
sure